### DISCUSSION TITLE
Welcome to stockDL Discussions!

### DISCUSSION THREAD

**Discussion by ashishpapanai** [ashishpapanai/stockDL#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BaseMax** [comment]
Hello Ashish
Is a trained model good for real investments?

I know most deep learning models are not suggested for real-life use. but Why did you not test the model to investment? If this has low error so can make $$$ for you.

**Reply by ashishpapanai** [reply]
Hey @BaseMax,

I appreciate the observations. The reason for not using the trained model in real life is the limitation. As correctly pointed out, the existing deep-learning model architecture is restricted to only one aspect or component of the financial market. In stockDL, we are focusing only on historical stock price data. 

I don't advise users to make financial decisions based on the predictions provided by the model (even with low losses), but working on a new model which simulates real-time stock trading (even as a game) and developing a reinforced agent on top of it would be a fantastic project/exercise to take-up and work on.

**Comment by Leci37** [comment]
My name is Luis, I'm a big-data machine-learning developer, I'm a fan of your work, and I usually check your updates.

I was afraid that my savings would be eaten by inflation. I have created a powerful tool that based on past technical patterns (volatility, moving averages, statistics, trends, candlesticks, support and resistance, stock index indicators).
All the ones you know (RSI, MACD, STOCH, Bolinger Bands, SMA, DEMARK, Japanese candlesticks, ichimoku, fibonacci, williansR, balance of power, murrey math, etc) and more than 200 others.

The tool creates prediction models of correct trading points (buy signal and sell signal, every stock is good traded in time and direction).
For this I have used big data tools like pandas python, stock market libraries like: tablib, TAcharts ,pandas_ta... For data collection and calculation.
And powerful machine-learning libraries such as: Sklearn.RandomForest , Sklearn.GradientBoosting, XGBoost, Google TensorFlow and Google TensorFlow LSTM.

With the models trained with the selection of the best technical indicators, the tool is able to predict trading points (where to buy, where to sell) and send real-time alerts to Telegram or Mail. The points are calculated based on the learning of the correct trading points of the last 2 years (including the change to bear market after the rate hike).

I think it could be useful to you, to improve, I would like to share it with you, and if you are interested in improving and collaborating I am also willing, and if not file it in the box.

If tou want, Please read the readme , and in case of any problem you can contact me ,
If you are convinced try to install it with the documentation.
https://github.com/Leci37/stocks-Machine-learning-RealTime-telegram/tree/develop I appreciate the feedback
